### ISSUE TITLE
[TF2] Numerous Items that drastically change the sillouette/head of a class aren't halloween restricted.

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Source-1-Games#6806]
This has been wildly talked to death before, but numerous cosmetic items that where introduced during the recent Scream fortress uptates weren't halloween restricted for reasons my mortal brain cannot fully comprehend. From complete shape changes to simply being out of this world when it comes to being annual.

Here is my personnal list of comsetics that should be restricted, without taking onto account any market value. Are people gonna hate me because I don't like how certain cosmetics aren't great outside of Halloween? Yes. But am I allowed to have an opinion. Yes.

Make Ogre: Literally turns heavy onto an Ogre yet isn't halloween restricted.
Defaced Detonator: Literally an half blowned face, nuff said.
Sixties Sidekick: innofensive enough, but feels more like an actual halloween costume than a serious thing.
Bare Bear Bones: A bear skull on the Heavy's face. Shouldn't this be halloween restricted.

**Comment by KimmyTF2** [comment]
Those are probably one of the least sillouette changing items. 
Make Ogre - pointy ears, nothing else changes his sillouette
Defaced Detonator - lietrally changes nothing, it covers only his face
Sixties Sidekick - a promo set, nothing completely out of place like many 2011-2012 promo items
Bare Bear Bones - it's just a skull, there are hats that are bigger than this

**Comment by DiskIntegrity** [comment]
I've personally never had a problem with a class' silhouette, because by the time I'm up close and personal to see their cosmetics I'm dead or about to die. Tbh, I find the whole argument dumb, similar to the "team recognition" argument as well.

There's a lot more Scream Fortress cosmetics like the Burly Beast and Catastrophic Companions _(and technically the Potassium Bonnet too since it was originally made for Halloween but was released in August for some reason)_ that could be restricted if Valve chose so but they wont because of the potential backlash.

**Comment by ghost** [comment]
I genually asked for restrictions because they are halloween items and thus makes no sense outside of this holyday. Because you know, that's the whole point of Halloween restriction in the first place? To allow the most wacky and insane stuff in with an actual justification even throught it makes absolutely no sense in the game's setting?

I know that 50% of players don't really give a damn about the out of place cosmetics that sometimes even break team recognition. But I simply wanted to point some things out. There's a reason people are fed up with hats to begin with.

The only real argument against my proposition is the market value of said items being diminished after a restriction and while it is completely justified. I just care more about the game's artstyle than money. But to each theirs own.

The irony is that there are many items who aren't restricted to Halloweem who can have the "Haunted" quality... I have no idea what else to say really.

**Comment by gdfsgdfg** [comment]
I agree with Defaced Detonator absolutely, the other ones mostly are "halloween costumes" at worst but yeah lets just have demoman with a half-exploded face. 

Even halloween restricted the double scout cosmetic is still the worst thing they have added in the game I wish it got replaced outright with a different cosmetic whole thing feels like you have a ragdoll that got stuck into your player model.
